src/main/java/com/example/bookstore/dto/BookRequestDto.java

```diff
@@ -1,11 +1,18 @@
 package com.example.bookstore.dto;
 
+import jakarta.validation.constraints.Digits;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+
 public record BookRequestDto(
+        @NotBlank
         String title,
+        @NotBlank
         String author,
+        @NotBlank
         String isbn,
+        @Min(0) @Digits(integer = 8, fraction = 2)
         String price,
         String description,
```

Comment:
You can validate description

Reply:
There is only one constraint about max length of description field in db.
Adding @NotBlank would make unavailable to update prices fow exsisting books with empty desciption. 
Here are a few options: 
Leave it as is.
Add @Length annotation
Update changeSet, Model and DTO

Reply:
You can just add `@Size `